src/main/java/org/zalando/nakadi/controller/EventPublishingController.java

```diff
@@ -99,6 +104,21 @@ private ResponseEntity postEventInternal(final String eventTypeName,
         }
     }
 
+    private void reportMetrics(final EventTypeMetrics eventTypeMetrics, final EventPublishResult result,
+                               final String eventsAsString, final int eventCount) {
+        if (result.getStatus() == EventPublishingStatus.SUBMITTED) {
+            eventTypeMetrics.reportSizing(eventCount, eventsAsString.getBytes(StandardCharsets.UTF_8).length - 2);
```

Comment:
We're looking for the average size of an event. They are sent as an array, so we should ignore the opening and closing square brackets.

Come to think of it, we should also ignore the commas between events. I'll fix that.